recipes/_environment.rb

```diff
@@ -21,7 +21,14 @@
 include_recipe 'omnibus::_common'
 
 if windows?
```

Comment:
Since we do not seem to have a unique variable in the generate scripts for the toolchain path, how do omnibus-software definitions such as Ruby determine the install path of toolchain for inclusion of dll files for runtime requirements?